array-manipulation-2/src/drop.js

```diff
@@ -0,0 +1,20 @@
+/* exported drop */
+
+// input: two parameters (array, count)
+// output: a new array
+
+// -- create a new variable of an empty array to store final array
+// -- loop through each item in array
+//     -- if the current index is greater than or euqal to 'count'
+//         -- push the item at current index into empty array
+// -- return final array
+
+function drop(array, count) {
+  var finalArray = [];
+  for (var i = 0; i < array.length; i++) {
+    if (i >= count) {
```

Comment:
If you start your loop at `count` rather than `0`, you can avoid this conditional